linux_os/guide/services/ssh/ssh_server/sshd_enable_warning_banner/rule.yml

```diff
@@ -36,6 +36,7 @@ references:
     srg: SRG-OS-000023-GPOS-00006,SRG-OS-000024-GPOS-00007,SRG-OS-000228-GPOS-00088
     vmmsrg: SRG-OS-000023-VMM-000060,SRG-OS-000024-VMM-000070
     stigid@rhel7: "040170"
+    stigid@sle12: 030050,030120
```

Comment:
There should be one rule here. Chose one, and then open a ticket at complianceascode/community saying that one that you didn't choose is a duplicate, that you are working with DISA to remove the duplicate item.